apps/playercontrol/app.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+var logger = require('logger')('@playercontrol')
+var fs = require('fs')
+var util = require('util')
+var DATAPATH = '/data/AppData/playercontrol/config.json'
+var DIRPATH = '/data/AppData/playercontrol'
+
+module.exports = function (activity) {
+  var STRING_NO_PLAYER_EXIST = '当前没在播放状态呢，如果你想听歌，请对我说，播放歌曲'
+
+  function speakAndExit (text) {
+    return activity.tts.speak(text).then(() => {
+      return activity.exit()
+    })
+  }
+
+  activity.on('request', function (nlp, action) {
+    logger.log('player control event: request', nlp)
+    if (nlp.intent === 'ROKID.INTENT.RESUME') {
+      readconfig(function (playerInfo, error) {
+        logger.log('have error ', error)
+        if (playerInfo === null || playerInfo === undefined || playerInfo === '') {
+          speakAndExit(STRING_NO_PLAYER_EXIST)
+          return
+        }
+        logger.log('name = ', playerInfo.name)
+        if (playerInfo.name === null || playerInfo.name === undefined || playerInfo.name === '') {
+          speakAndExit(STRING_NO_PLAYER_EXIST)
+        } else if (playerInfo.name === '@yoda/bluetooth') {
+          activity.voiceCommand('播放蓝牙音乐')
```

Comment:
activity.openUrl('yoda-skill://bluetooth_music/bluetooth_start_bluetooth_music', 'scene')
Can directly openUrl to start bluetooth music. No need mock voice command due to it may take too much time.

Reply:
+1 on @morrowind's suggestion, voiceCommand takes more time.